### PR TITLE
VZ-8859 Adding tiles to vz homepage

### PR DESCRIPTION
**Pull request by stevshap**
This PR adds outlined tiles as shown below:
Screenshot 2023-03-15 at 4 36 50 PM

This PR is not intended to address other components of the home page

### CODE REVIEW

**Comment by gilbode** [comment]
https://www.docsy.dev/docs/adding-content/shortcodes/#blockssection - set color on the sction shortcode

File: assets/scss/_variables_project.scss [hunk 1]
```diff
@@ -60,4 +60,23 @@ $lead-font-weight: 400 !default;
 .td-content > h7 { font-weight: 700; }
 
 // Custom CSS for homepage
+.light-shadow {
```
Comment (gilbode) [inline assets/scss/_variables_project.scss:63]: What is the purpose of all these CSS changes?

**Comment by gilbode** [comment]
> The white text over the background image is a little hard to read

Setting color=light on the cover shortcode will make the text dark, but that's only marginally better for the lighter font weights.

File: assets/scss/_variables_project.scss (hunk 1 above)
Comment (knelson7) [reply assets/scss/_variables_project.scss:63]: This change is styling the new element "tile" that he created

Comment (gilbode) [reply assets/scss/_variables_project.scss:63]: I see - tile reads too close to title to me.

**Comment by rosemarymarano** [comment]
A few comments/feedback from the doc team:

- In the upper right corner, the Documentation text/link is almost invisible, hard to see/find.
- One way to increase the visibility of the white text, "A hybrid multicloud Kubernetes..." would be in encircle it in a dark blue oval similar to what is done for Learn More and View Repository.
- A suggestion to make the vertical space from the first to the second set of boxes, the same as from diagram to the first set of boxes.

**Comment by gilbode** [comment]
We can make the entire navbar have a dark background instead of having it translucent w/ the image.

**Comment by stevshap** [comment]
Updated UI:
Screenshot 2023-03-15 at 4 36 50 PM

**Review by david-cabelus-oracle — APPROVED** [review]

**Pull request merged**